### ISSUE TITLE
Slow on very large files (with vim plugin?)

### ISSUE THREAD

**Issue by jpmorris** [cmoog/vscode-sql-notebook#64]
I am noticing nearly 0.5 to 1 second keypress lags with SQL NOTEBOOKS for my 2000-line file.  I have another file that is only 275 lines that I don't see this, and I don't see this in 1000-line python files.   It appears that it goes away if the vim plugin is disabled.   Not sure if the issue is related to this plugin, but figured you should know.

**Comment by cmoog** [comment]
Thank you for this report. I have yet to reproduce this behavior. I tested a 3,000-line SQL file opened as a notebook with Vim enabled and saw no increase in input latency. The same was true for a 3,000-line .ipynb file opened as a notebook. I also enabled and disabled the embedded language server configuration option, with no perceivable effect. I also tried with local files vs. remote files (via Remote SSH) and saw no difference.

Any other ideas about what might be different with your setup?
